docs/pf-keys/tutorials/solid_files.rst

```diff
@@ -87,7 +87,7 @@ Full API: SolidFileBuilder
 
 2. ``mask(mask_array)`` - applies the matrix array ``mask_array`` to the SolidFileBuilder object.
 
-3. ``write(self, name, xllcorner=0, yllcorner=0, cellsize=0, vtk=False, extra=None)`` - writes the ``SolidFileBuilder`` object data to the *.pfsol* file ``name``. The arguments ``xllcorner``, ``yllcorner``, and ``cellsize=0`` help define the size of the solid file domain. If ``vtk`` is set to ``True``, it will write a VTK file ``name.vtk`` that you can view in ParaView or another VTK viewer to check that the solid file is correct. If there are any extra arguments you want to pass to the ``pfmask-to-pfsol`` converter in Parflow, specify them using the ``extra`` parameter, as a list of strings.
+3. ``write(self, name, xllcorner=0, yllcorner=0, cellsize=0, vtk=False, extra=None, generate_asc_files=True)`` - writes the ``SolidFileBuilder`` object data to the *.pfsol* file ``name``. The arguments ``xllcorner``, ``yllcorner``, and ``cellsize=0`` help define the size of the solid file domain. If ``vtk`` is set to ``True``, it will write a VTK file ``name.vtk`` that you can view in ParaView or another VTK viewer to check that the solid file is correct. If there are any extra arguments you want to pass to the ``pfmask-to-pfsol`` converter in Parflow, specify them using the ``extra`` parameter, as a list of strings. By default, this method generates .asc files for top/bottom/sides and calls ``pfmask-to-pfsol`` with individual 'mask-*' flags for top/bottom/sides, each pointing to a .asc file. Setting ``generate_asc_files`` to ``False`` calls ``pfmask-to-pfsol`` with a single 'mask' flag pointing to a temporary .pfb mask file.
```

Comment:
Would be helpful for docs to indicate what the filenames of the ASC files will be.

I don't think the temporary pfb file is important to users.  If I understand generate_asc_files simply controls if the ASC files are created or not in terms how user uses the method.

Reply:
True - made the change